src/jdk.compiler/share/classes/com/sun/tools/javac/model/AnnotationProxyMaker.java

```diff
@@ -220,7 +220,7 @@ public void visitArray(Attribute.Array a) {
             }
         }
 
-        @SuppressWarnings({"unchecked", "rawtypes"})
+        @SuppressWarnings("unchecked")
```

Comment:
Tricky but correct :-) This relies on the special pleading for casts to "raw" `Class`